content/20.treatments.md

```diff
@@ -115,6 +115,12 @@ Remdesivir has also been reported to inhibit SARS-CoV-2 infection in a human cel
 The effectiveness of Remdesivir for treating patients with COVID-19 is currently under investigation.
 Remdesivir has been used on some COVID-19 patients under compassionate use guidelines [@doi:10.1126/science.abb7243, @doi:10.1056/NEJMoa2001191; @doi:10.1101/2020.03.09.20032896].
 All were in late stages of COVID-19 infection, and these reports are inconclusive about the drug's efficacy.
+Indeed, the New England Medical Journal published a Gilead Sciences sponsored trial that administered 200mg of Remdesivir intravenously on day 1 followed by a further 100mg/day for 9 days on compassionate-use basis to 61 patients hospitalized with confirmed COVID-19 [@doi:10.1056/NEJMoa2007016].
+Following treatment, data from 53 patients were analyzed and a clinical improvement was observed in 68% (36/53) of patients.
+While that result might seem promising, there were significant issues with the study design that undermine the seemingly positive outcome.
+These included the fact that the trial did not include a randomized control group, some patients only required low doses of oxygen, others required ventilation, patients analyzed had mixed demographics as it was a multisite trial, there was a short follow-up period of investigation, some patients worsened, some patients died, and lastly 8 patients were excluded from the analysis mainly due to missing post-baseline information, thus their health is unaccounted for.
+As a result of these confounding issues, it is difficult to determine if there was any treatment effect or would these outcomes have occurred regardess of treatment.
+Therefore, no conclusive deductions can be made at this time until  randomised controlled trials are completed.
```

Comment:
```suggestion
Therefore, the study does not alter our understanding of the efficacy of Remdesivir in this setting, and randomized controlled trials are still required.
```

Reply:
If you take my suggestion on the previous sentence, here I would suggest using Casey's sentence but swapping "Therefore" with "As a result"